README.md

```diff
@@ -57,7 +60,7 @@ You can propose popular transformer rules to be included in the plugin by [sugge
 
 **How do I post articles to Instant Articles after plugin is installed?**
 
-You can re-publish existing articles (simply edit + save) or post new articles in order to submit them to Instant Articles. After you have 10 articles added, you will be able to submit them for review.
+You can re-publish existing articles (simply edit + save) or post new articles in order to make them available as Instant Articles. After you have 5 articles added and shared, you will be able to submit them for review.
```

Comment:
I think now it's 10 indeed: https://developers.facebook.com/docs/instant-articles/quickstart#review

In fact, now you don't need to edit+save your posts to submit - it doesn't work anymore. We should let them know that new articles will be submitted when shared and old articles can be re-scraped through the Share Debugger.